wrims/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+repositories {
+    mavenLocal()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+sourceSets {
+    src {
+        java {
+            srcDirs "src/main/java/"
+        }
+    }
+}
+
+dependencies {
+    implementation project(':wrims-core')
+
+    implementation libs.commons.io
+    implementation libs.jfreechart
+    implementation libs.hec.monolith
+    implementation libs.hec.nucleus.metadata
+
+    implementation fileTree(dir: 'libs/eclipse', include: ['*.jar'])
+    implementation fileTree(dir: 'libs/x64', include: ['*.jar'])
+    implementation fileTree(dir: 'libs/x64', include: ['*.dll'])
+
+    // The following dependencies are used to build the OSGI bundle
+    implementation 'org.osgi:org.osgi.core:6.0.0'
```

Comment:
Add to version catalog (libs.versions.toml)

Reply:
Added to version catalog